pipeline/lint.bash

```diff
@@ -4,13 +4,15 @@ set -eu -o pipefail
 
 here="$(dirname "$(readlink -f "$0")")"
 
-# TODO: Try to get rid of the NFS server IP.
-# We need to set the NFS server IP on Exoscale for the nfs-provisioner.
-NFS_SC_SERVER_IP=127.0.0.1 helmfile -e service_cluster -f "${here}/../helmfile/" lint
-NFS_WC_SERVER_IP=127.0.0.1 helmfile -e workload_cluster -f "${here}/../helmfile/" lint
+#We only lint our "own" charts, not the upstream ones
+charts_ignore_list="app!=ck8sdash,app!=sc-logs-retention,app!=fluentd-configmap,app!=ingress-nginx,app!=cert-manager,app!=kube-prometheus-stack,app!=velero,app!=metrics-server,app!=dex,app!=wc-scraper,app!=prometheus-auth,app!=wc-reader,app!=prometheus-wc-reader,app!=user-grafana,app!=prometheus-elasticsearch-exporter,app!=harbor,app!=influxdb,app!=fluentd,app!=fluentd-aggregator,app!=opendistro,app!=falco,app!=falco-exporter,app!=user-alertmanager,app!=kubeapi-metrics"
```

Comment:
To make it easier to get a grasp of what applications we are ignoring I suggest:
```suggestion
charts_ignore_list=(
  app!=ck8sdash
  app!=sc-logs-retention
  app!=fluentd-configmap
  app!=ingress-nginx
  app!=cert-manager
  app!=kube-prometheus-stack
  app!=velero
  app!=metrics-server
  app!=dex
  app!=wc-scraper
  app!=prometheus-auth
  app!=wc-reader
  app!=prometheus-wc-reader
  app!=user-grafana
  app!=prometheus-elasticsearch-exporter
  app!=harbor
  app!=influxdb
  app!=fluentd
  app!=fluentd-aggregator
  app!=opendistro
  app!=falco
  app!=falco-exporter
  app!=user-alertmanager
  app!=kubeapi-metrics )
```

Reply:
Good idea, have changed to this implementation.